### ISSUE TITLE
warning message: CRS object has comment, which is lost in output

### ISSUE THREAD

**Issue by adokter** [adokter/bioRad#456]
```
project_as_ppi(example_scan)
               Plan position indicator (class ppi)

  parameters:  VRADH DBZH ZDR RHOHV PHIDP 
        dims:  201 x 200 pixels

Warning message:
In proj4string(obj) : CRS object has comment, which is lost in output
```
This warning message disappears when upgrading to the latest development version of sp

See https://rgdal.r-forge.r-project.org/articles/CRS_projections_transformations.html

**Comment by adokter** [comment]
fix is part of #505

**Comment by adokter** [comment]
If these options are set before the user loads rgdal or sp, the warnings disappear
```
options("rgdal_show_exportToProj4_warnings"="none") 
```
